src/hotspot/share/gc/shenandoah/shenandoahLock.cpp

```diff
@@ -44,7 +44,7 @@ void ShenandoahLock::contended_lock(bool allow_block_for_safepoint) {
 template<bool ALLOW_BLOCK>
 void ShenandoahLock::contended_lock_internal(JavaThread* java_thread) {
   assert(!ALLOW_BLOCK || java_thread != nullptr, "Must have a Java thread when allowing block.");
-  // Spin this much on multi-processor, do not spin on multi-processor.
+  // Spin this much on multi-processor, do not spin on single-processor.
```

Comment:
I think the proper nomenclature is "uniprocessor". We can avoid all this if we say e.g. "// Spin this much, but only on multi-processor systems"

Reply:
Thanks, suggestion is taken, I updated the PR.